tools/clang/test/CodeGenSPIRV/meshshading.nv.error.fncall.amplification.vulkan1.2.hlsl

```diff
@@ -1,8 +1,8 @@
-// RUN: %dxc -T as_6_5 -E main -fspv-target-env=vulkan1.2
+// RUN: %dxc -T as_6_5 -E main -fspv-target-env=vulkan1.2 -fcgl -spirv %s | FileCheck %s
 
-// CHECK:  OpCapability MeshShadingNV
-// CHECK:  OpExtension "SPV_NV_mesh_shader"
-// CHECK:  OpEntryPoint TaskNV %main "main"
```

Comment:
Why did you remove the "CHECK"?

Reply:
Oops, totally unintentional. Good catch.